src/main/cpp/chassis/SwerveModule.cpp

```diff
@@ -260,6 +269,21 @@ void SwerveModule::InitDriveMotor(bool driveInverted)
     {
         m_driveTalon->GetConfigurator().Apply(TalonFXConfiguration{}); // Apply Factory Defaults
 
+        configs::TalonFXConfiguration configs{};
+
+        configs.Voltage.PeakForwardVoltage = 11;
+        configs.Voltage.PeakReverseVoltage = -11;
+
+        /* Torque-based velocity does not require a feed forward, as torque will accelerate the rotor up to the desired velocity by itself */
+        /// TO DO : Need code gen updates to be able to be implemented
+        configs.Slot1.kS = 2.5;       // To account for friction, add 2.5 A of static feedforward
+        configs.Slot1.kP = m_driveKp; // An error of 1 rotation per second results in 5 A output
+        configs.Slot1.kI = m_driveKi; // No output for integrated error
+        configs.Slot1.kD = m_driveKd; // No output for error derivative
+        // Peak output of 40 A
```

Comment:
Should some of these settings be dependent upon the control mode?

Reply:
Sorry what do you mean by that? LIke if it is Velocity Voltage controlled or if it should be Torque controlled?